rfcs/text/0000-extend-file-pe.md

```diff
@@ -0,0 +1,142 @@
+# 0000: Extend the PE field set
+
+- Stage: **1 (proposal)**
+- Date: **TBD**
+
+The Portable Executable (PE) sub-field, of the `file` top-level fieldset, can be updated to include more file attributes to aid in file analysis. This additional document metadata can be used for malware research, as well as coding and other application development efforts.
+
+## Fields
+
+This RFC is to create 25 additional sub-fields within the `file.pe` fieldset.
```

Comment:
This includes updates to these as well, right?
* `dll.pe`
* `process.pe`

Reply:
`pe.*` is nested under `dll.*` and `process.*`. Any updates applied to the top-level field set will also follow under the nested uses.